src/web/services/apiKeyService.ts

```diff
@@ -56,6 +56,6 @@ export async function DisableApiKey(apiKey: ApiKeyDTO, participantId?: number) {
       data: { keyId: apiKey.key_id },
     });
   } catch (e: unknown) {
-    throw backendError(e, 'Could not disable API Key');
```

Comment:
Is this something a user would see? If not, I think it makes sense to leave as disable since that is what is happening from an engineer's point of view, but if the user would see this, then I'm good with the change to 'delete'

Reply:
Yes this flows through to the UI